serial_communication/web/app.py

```diff
@@ -62,8 +85,9 @@ def send_serial():
 def send_auth_route():
     return send_auth()
 
-schedule.every(2).minutes.do(update_time)
-schedule.every(1).minutes.do(update_namaz_time)
+if bg_tasks: 
+    schedule.every(2).minutes.do(update_time) 
+    schedule.every(5).minutes.do(update_namaz_time) #TODO: implement array
```

Comment:
for good practice I think we should also write code for else part so we at least know if this part interrupt the normal flow of code.

Reply:
yes, I think we don't need now but may be in future when we take over the unknow message control.